tests/chem_dm_kBE_test.py

```diff
@@ -9,16 +9,12 @@
 import os
 import unittest
 
+import libdmet
```

Comment:
If we are always importing `libdmet` now, these decorators in the rest of this file below can go away, right?
```
@unittest.skipIf(libdmet is None, "Module `libdmet` not imported correctly.")
```

Reply:
good catch thank you!